<!--
ROB599 Homwork 3 - timer node sends messages to itself.
Jake Anderson
Oct 24, 2020
-->


<launch>

	<!-- Start the timer node. -->
	<remap from="/timer/queued_scan" to="/timer/clocked_scan"/>
	<node pkg="rob599_hw3" name="laser_timmer" type="timer_node" output="screen" >
	</node>


</launch>
